.github/workflows/release_notes_check.yml

```diff
@@ -0,0 +1,25 @@
+name: Release Notes Check
+
+on:
+  pull_request:
+    types:
+      # On by default if you specify no types.
+      - "opened"
+      - "reopened"
+      - "synchronize"
+      # For `skip-label` only.
+      - "labeled"
+      - "unlabeled"
+
+jobs:
+  check-release-notes:
+    name: Check release notes are updated
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check for a release notes update
+        uses: brettcannon/check-for-changed-files@294a99714e0d350b5083472a293d41bc91804e68
```

Comment:
Can you add a comment with the tag or branch being used? I think it's needed for dependabot to take action. See https://github.com/frequenz-floss/frequenz-sdk-python/blob/v0.x.x/.github/workflows/labeler.yml#L21 for example.